src/main/java/pl/common/todo/api/ToDoApi/model/Task.java

```diff
@@ -2,23 +2,30 @@
 
 import java.util.Date;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
 public class Task {
-	
-	private Integer id;
-	
+
+	@Id
+	@GeneratedValue
+	private Long id;
+
 	private String name;
-	
+
 	private Date deadline;
-	
-	private Date insertionDate;
-	
-	private Date updateDate;
 
-	public Integer getId() {
+	private Date insertionDate = new Date();
+
+	private Date updateDate = new Date();
```

Comment:
Do bazy musi być wrzucana jednocześnie tylko jedna z tych dwóch dat. Kiedy wrzucamy nowe zadanie to nie jest ono od razu aktualizowane przecież co nie ?

Reply:
Usunąłem domyślne ustawianie obydwu dat. Faktycznie masz rację :)